lib/models/NotificationConfiguration.js

```diff
@@ -289,21 +289,22 @@ class NotificationConfiguration {
      */
     static validateConfig(config) {
         assert.strictEqual(typeof config, 'object');
-        assert.strictEqual(Array.isArray(config.queueConfig), true);
+        if (!config.queueConfig) {
+            return;
+        }
         config.queueConfig.forEach(q => {
             const { events, queueArn, filterRules, id } = q;
-            assert.strictEqual(Array.isArray(events), true);
             events.forEach(e => assert.strictEqual(typeof e, 'string'));
             assert.strictEqual(typeof queueArn, 'string');
             if (filterRules) {
-                assert.strictEqual(Array.isArray(filterRules), true);
                 filterRules.forEach(f => {
                     assert.strictEqual(typeof f.name, 'string');
                     assert.strictEqual(typeof f.value, 'string');
                 });
             }
             assert.strictEqual(typeof id, 'string');
         });
+        return;
```

Comment:
I'm surprised this is needed, if it's linter complaint, fair enough

Reply:
yeah, just for the linter